docs/pages/installation.rst

```diff
@@ -22,6 +22,22 @@ Install the package from the ``conda-forge`` channel:
 .. code-block:: bash
 
     conda install dask-sql -c conda-forge
+ 
+With ``GPU Support``
+--------------
+Create a new conda environment or use an existing one to install RAPIDS with the chosen methods and packages:
```

Comment:
I think we should maybe add a short blurb about GPU support here, bringing up the following to contextualize the env creation line below:

- GPU support right now is specifically tied to the [RAPIDS](https://rapids.ai/) libraries
- It is in development, and generally requires the latest nightlies of cuDF
- It is experimental, and users should still expect some level of bugs / undefined behavior

Reply:
Thanks @charlesbluca, I'll see to add them!